packages/steppp-react/src/components/Provider.tsx

```diff
@@ -11,16 +11,16 @@ interface ProviderProps {
 
 const baseCss = `
     [data-steppp-wrapper] {
-        position: relative;
-        overflow: hidden;
+      position: relative;
+      overflow: hidden;
     }
 
-    [data-steppp-active] {
-        display: block;
+    [data-steppp-name] {
```

Comment:
Here's the gist. Instead of hiding any "section" on the page, it only hides those with the data-steppp-name attribute.